code/src/pages/MovieList.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react'
+
+import { Link } from 'react-router-dom'
+import Loader from '../components/Loader'
```

Comment:
Again I believe the space should be between the last import from the library and your first component.